.travis.yml

```diff
@@ -31,7 +32,7 @@ before_install:
         fi
 
 install:
-    - composer update $COMPOSER_FLAGS $PREFER_LOWEST
+    - composer update --ignore-platform-req=php $COMPOSER_FLAGS $PREFER_LOWEST
```

Comment:
This can be reverted as soon as `friendsofphp/php-cs-fixer` is happy to install on PHP 8.0